sys-utils/unshare.c

```diff
@@ -443,6 +478,14 @@ int main(int argc, char *argv[])
 		case 'w':
 			newdir = optarg;
 			break;
+                case OPT_MONOTONIC:
+			monotonic = strtoul_or_err(optarg, _("failed to parse uid"));
```

Comment:
It seems like copy & past mistake. You probably want "failed to parse monotonic offset".